templates/components/progress-circle.tpl

```diff
@@ -48,14 +46,12 @@
     class="progress-ring__circle"
     stroke="currentColor"
     stroke-width="{$stroke}"
-    data-percent="50"
+    data-percent="0"
     fill="transparent"
     r="{$radius}"
     cx="{$size / 2}"
     cy="{$size / 2}"
-    style="stroke-dasharray: {$circumference}, {$circumference}; stroke-dashoffset: {$dashoffset};"
+    style="stroke-dasharray: {$circumference}, {$circumference}; stroke-dashoffset: {$circumference};"
```

Comment:
Are you sure? It's working fine?

Reply:
In initial state (0%) the circle should be gray.
{$dashoffset = $circumference - (($percent / 100 ) * $circumference)}
so when $percent is 0 => $dashoffset = $circumference